src/app/project-overview/component.ts

```diff
@@ -190,4 +195,14 @@ export class ProjectOverviewComponent implements OnInit, OnDestroy {
       .pipe(takeUntil(this._unsubscribe))
       .subscribe(serviceAccounts => (this.serviceAccounts = serviceAccounts));
   }
+
+  private _checkFirstVisitToOverviewPageMessage(): void {
+    this._cookieService.get(this._cookieName)
+      ? this.hideFirstVisitToOverviewPageMessage()
+      : this._cookieService.set(this._cookieName, 'visited', null, '/', 'localhost', false, 'Lax');
```

Comment:
I guess this won't work on Dev since `hardcode` binding of **Domain** it should be `window.location.hostname` 

cc: @Waseem826